include/nfsc/libnfs-raw.h

```diff
@@ -24,6 +24,7 @@
 #define _LIBNFS_RAW_H_
 
 #include <stdint.h>
+#include <sys/uio.h>
```

Comment:
Will make this conditional for Windows and other non posix compliant platforms that lack readv()/writev() calls.